src/modules/home/components/category/category-item.tsx

```diff
@@ -0,0 +1,53 @@
+import Typography from '@mui/material/Typography';
+import { type CategoryWithIcon } from '../../types/category-with-icon.type';
+import Box from '@mui/material/Box';
+
+const categoryItemContainerStyles = {
+  display: 'flex',
+  flexDirection: 'column',
+  justifyContent: 'center',
+  alignItems: 'center',
+  paddingX: 4,
+  color: 'gray',
+  paddingY: 2,
+  '&:hover': {
+    backgroundColor: 'rgba(0, 0, 0, 0.04)',
+    color: '#000',
```

Comment:
Avoid just color codes. If its commonly used color, create a constant with the name for that color and use that.